utils/make_release_notes.sh

```diff
@@ -76,6 +78,15 @@ CYCAMORETXT="$NCOMMITS commits resulting in $SUMMARY"
 CYCAMORECONTRIB=`git log --format="%aN" $PREV...$VERSION | sort -u`
 cd $HERE
 
+# cymetric summary
+cd $CYMETRIC
+[[ `git tag` =~ $PREV ]] && CYMETRIC_PREV=$PREV || CYMETRIC_PREV=`git log --pretty=format:%H|tail -1`
```

Comment:
I don't know how to configure it properly to run (because I still don't understand bash) - how do I set the directories?